handler/src/main/java/io/netty/handler/ssl/OpenSslContextOption.java

```diff
@@ -68,4 +68,10 @@ private OpenSslContextOption(String name) {
      */
     public static final OpenSslContextOption<OpenSslCertificateCompressionConfig> CERTIFICATE_COMPRESSION_ALGORITHMS =
             new OpenSslContextOption<OpenSslCertificateCompressionConfig>("CERTIFICATE_COMPRESSION_ALGORITHMS");
+
+    /**
+     * Set the maximum number of bytes that is allowed during the handshake for certificate chain.
+     */
+    public static final OpenSslContextOption<Long> MAX_CERTIFICATE_LIST =
```

Comment:
`MAX_CERTIFICATE_LIST` -> `MAX_CERTIFICATE_LIST_BYTES` to clarify it limits total bytes?

I assume JDK does not have any API to limit max size in bytes of cert chain?

Reply:
+1